examples/ola-rdm.cpp

```diff
@@ -207,16 +207,22 @@ void DisplayHelpAndExit(const options &opts) {
   exit(ola::EXIT_USAGE);
 }
 
-
 /*
  * Dump the list of known pids
  */
 void DisplayPIDsAndExit(uint16_t manufacturer_id,
-                        const PidStoreHelper &pid_helper) {
+                        const PidStoreHelper &pid_helper,
+                        bool set_mode) {
+  vector<const ola::rdm::PidDescriptor *> pids;
+  pid_helper.SupportedPids(manufacturer_id, &pids);
+  // Remove PIDs that don't support the proper command class (GET with ola_rdm_get, SET with ola_rdm_set)
   vector<string> pid_names;
-  pid_helper.SupportedPids(manufacturer_id, &pid_names);
-  sort(pid_names.begin(), pid_names.end());
```

Comment:
You seem to have dropped the sort in the rewrite. It would be good to get that back, either as a sort of the strings you're accumulating, or perhaps create a sort function (if we don't have one already), sort the descriptors beforehand then print directly.